react/src/components/ResourcePresetList.tsx

```diff
@@ -103,7 +103,9 @@ const ResourcePresetList: React.FC<ResourcePresetListProps> = () => {
       title: t('resourcePreset.SharedMemory'),
       dataIndex: 'shared_memory',
       render: (text) =>
-        text ? convertBinarySizeUnit(text + '', 'g')?.numberFixed : '-',
+        text
+          ? convertBinarySizeUnit(text + '', 'g', 3, true)?.numberFixed
```

Comment:
```suggestion
          ? convertBinarySizeUnit(text + '', 'g', 2, true)?.numberFixed
```
The Lablup UX Writing Guideline states that shared memory supports up to 2 units.

Reply:
해당 가이드에 맞게 수정하면 값이 0으로 표현되는데 이건 괜찮나요?

![CleanShot 2025-04-22 at 13.31.57@2x.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/xbCemO1RqqcSjEXCRK3p/dfb12891-8118-49b4-9df2-6eb6c39febc1.png)